### ISSUE TITLE
Add ResNet-RS performance to https://keras.io/api/applications/

### ISSUE THREAD

**Issue by sayakpaul** [keras-team/keras-io#908]
The purpose of this thread is to just request the contributor (@spatil6) of the model to add the following details and contribute a PR to https://keras.io/api/applications/:

* Size
* Top-1 Accuracy
* Top-5 Accuracy
* Parameters
* Depth (if applicable) 

The rest of the details (inference timings) can be still avoided since the hardware might vary (the official experiments were conducted on a specific machine configuration which may not be available to contributor). 

Cc: @LukeWood @fchollet @qlzh727

**Comment by spatil6** [comment]
Sure @sayakpaul , will raise PR soon!
